pyblish_qml/control.py

```diff
@@ -1046,4 +1052,14 @@ def iterator(plugins, context):
                 yield plugin, instance
 
         else:
+        
+     
+            # When filtering to families at least a single instance with
+            # that family must be active in the current publish
+            if "*" not in plugin.families:
+                if not any(instance.data.get("publish") is not False 
```

Comment:
I can't make sense of this line.

If not <any of these> is not <Not True>, then loop through the below. Have a look and see if you can't make this more intuitive

Reply:
The check is this explicit to check for `False` because I believe the Pyblish logic does the same. As such I couldn't just do `if not instance.data.get("publish")` because then it would also consider `None` whereas the previous Pyblish logic wouldn't.

The logic is that _at least one instance must have its publish set to True_. So if there's no "active" one it should `continue`.

The check whether it's true is: `instance.data.get("publish") is not False`

This could also work:
```python
def _is_active(instance):
    """Whether instance is currently set active to publish"""
    return instance.data.get("publish") is not False 

if not any(_is_active(instance) for instance in instances):
    continue
```